crates/control/src/behavior/calibrate.rs

```diff
@@ -6,20 +6,17 @@ use types::{
 };
 
 pub fn execute(world_state: &WorldState) -> Option<MotionCommand> {
-    match world_state.robot.primary_state {
-        PrimaryState::Calibration => {
-            let head_motion = match world_state.calibration_command {
-                Some(CalibrationCommand::LookAt { target, camera, .. }) => HeadMotion::LookAt {
-                    target,
-                    camera: Some(camera),
-                    image_region_target: ImageRegion::Bottom,
-                },
-                // TODO Add walk-to-penalty area/ centre circle during CalibrationCommand::INITIALIZE
-                _ => HeadMotion::Center,
-            };
-
-            Some(MotionCommand::Stand { head: head_motion })
-        }
-        _ => None,
-    }
+    let (PrimaryState::Calibration, Some(CalibrationCommand::LookAt { target, camera, .. })) = (
+        world_state.robot.primary_state,
+        world_state.calibration_command.clone(),
```

Comment:
You don't need to clone here if you make the `CalibrationCommand` `Clone`.

Reply:
Done